sycl/doc/extensions/README.md

```diff
@@ -27,7 +27,7 @@ DPC++ extensions status:
 | [SYCL_INTEL_reqd_work_group_size](ReqdWorkGroupSize/SYCL_INTEL_reqd_work_group_size.asciidoc)                               | Supported(OpenCL: CPU, GPU)               | |
 | [SPV_INTEL_function_pointers](SPIRV/SPV_INTEL_function_pointers.asciidoc)                                                   | Supported(OpenCL: CPU, GPU; HOST)         | |
 | [SPV_INTEL_inline_assembly](SPIRV/SPV_INTEL_inline_assembly.asciidoc)                                                       | Supported(OpenCL: GPU)                    | |
-| [SYCL_INTEL_local_memory](LocalMemory/SYCL_INTEL_local_memory.asciidoc)                                                     | Proposal                                  | |
+| [SYCL_INTEL_local_memory](LocalMemory/LocalMemory.asciidoc)                                                                 | Proposal                                  | |
```

Comment:
This should now be `[SYCL_EXT_ONEAPI_LOCAL_MEMORY](...)` right?

Reply:
Good catch!  Thanks, I'll fix this now.

Reply:
Fixed in 577bf0d805085ebdeb3dd32c3063eb56e7cac24c.  I realized that the more recent extensions (e.g. Uniform, InvokeSIMD) just have their name without the SYCL_EXT_ONEAPI part, so I changed this to read "LocalMemory".

Reply:
Sounds good! Thanks.